.pre-commit-config.yaml

```diff
@@ -6,15 +6,17 @@ repos:
     rev: v0.1.2
     hooks:
       - id: ruff
-        args: ["--config", "pyproject.toml"]
+        args:
+          - --config=pyproject.toml
         exclude: "{{cookiecutter.project_slug}}/src/{{cookiecutter.package_name}}"
       - id: ruff-format
-        args: ["--config", "pyproject.toml"]
+        args:
+          - --config=pyproject.toml
   - repo: https://github.com/codespell-project/codespell
     rev: v2.2.6
     hooks:
       - id: codespell
-        args: [--write-changes]
+        exclude: ".lock$"
```

Comment:
@samcunliffe can you please look at this line

Reply:
pre-commit will still cause the linting to fail in the case of a false positive.

Reply:
Yes, but we can exclude that file in that case.

Reply:
So we just continually add exceptions? Maybe codespell is loads better now. It's certainly actively maintained.

I still vote that it should be dropped entirely from pre-commit and just advertised as a recommended tool.

Reply:
I do think you are talking a very niche case based on prior bad experience.

For the case of this repo (not the cookie generated one), I think it's only the `Gemfile.lock` that is mission-critical.

As I said I've added to the MIRSG general hooks, where we just exclude certificate files for everything https://github.com/UCL-MIRSG/.github/blob/35ec8a3074aa34903640bba9b78753ad43159b1e/precommit/mirsg-hooks.yaml#L1-L42

Reply:
I'm fine with being overruled if that's the majority decision.

Reply:
Turns out this was already being done in https://github.com/UCL-ARC/python-tooling/blob/d96dd24fbd8dfe719757b3e19dae79a36ae9086a/pyproject.toml#L1-L2 and I'm assuming the PR in question merged off `main` before that entered